WordPress/src/main/java/org/wordpress/android/ui/layoutpicker/LayoutsRowViewHolder.kt

```diff
@@ -27,7 +29,7 @@ class LayoutsFooterViewHolder(parent: ViewGroup, footerLayoutResId: Int) :
 class LayoutsItemViewHolder(
     parent: ViewGroup,
     private val prefetchItemCount: Int = 4,
-    private val showRowDividers: Boolean,
+    private val isPageLayoutCategory: Boolean,
```

Comment:
This is just a suggestion and more of a personal preference 🙏 
Renaming the `showRowDividers` to `isPageLayoutCategory` has the advantage of simplifying the code. On the other hand the naming implies that the parent implementation has knowledge of the specifics of one of the children. Maybe an alternative approach (with the disadvantage of needing more code) would be to add something like an `useLargeCategoryHeading: Boolean` property. Wdyt?

Reply:
Makes sense to me, this way we maintain the loose coupling 👍

Reply:
Addressed in [cefc268](https://github.com/wordpress-mobile/WordPress-Android/pull/16619/commits/cefc268622e35d854152f76608f380346e0ec141)

Reply:
Thank you for the change @ovitrif 🙇 
The code looks good. I've also retested with the latest changes and everything works as expected 🎉